pkg/trace/sampler/memory_backend.go

```diff
@@ -16,6 +17,9 @@ import (
 // Its bias with steady counts is 1 * decayFactor.
 // The stored scores represent approximation of the real count values (with a countScaleFactor factor).
 type MemoryBackend struct {
+	// seen, kept are used for telemetry
+	seen int64
+	kept int64
 	// scores maps signatures to scores.
 	scores map[Signature]float64
```

Comment:
If you could improve the documentation for all of this struct, making it easy to understand, it would serve as a great backbone to understanding all samplers.

Reply:
I will try to do that on another PR, I need to think a bit about how to make this simpler